tests/test_elastic.py

```diff
@@ -18,6 +18,11 @@
     UnregUserFactory, UnconfirmedUserFactory
 )
 
+TEST_INDEX = 'test'
+elastic_search.INDEX = TEST_INDEX
+settings.ELASTIC_INDEX = TEST_INDEX
```

Comment:
We should avoid import side-effects if possible; any reason this couldn't be done in `SearchTestCase#setUp`?